packet.c

```diff
@@ -2160,6 +2176,133 @@ ssh_packet_disconnect(struct ssh *ssh, const char *fmt,...)
 	cleanup_exit(255);
 }
 
+#ifdef MPTCP_GET_SUB_IDS
+
```

Comment:
According to the OpenBSD style guidelines, I think those methods should be declared static and prototype should be put at the top of the file